src/utils/platformServices.js

```diff
@@ -0,0 +1,9 @@
+const authenticateUser = () => {
```

Comment:
Add tests for this method.  Examples [here](https://github.com/RedHatInsights/subscription-inventory-ui/blob/44b24d6e84aeab975f4457fb8aa8888aa67ea9b6/src/utilities/__tests__/platformServices.test.ts#L19).